modules/ingest-common/src/test/java/org/opensearch/ingest/common/RemoveProcessorTests.java

```diff
@@ -67,12 +66,24 @@ public void testRemoveNonExistingField() throws Exception {
         config.put("field", fieldName);
         String processorTag = randomAlphaOfLength(10);
         Processor processor = new RemoveProcessor.Factory(TestTemplateService.instance()).create(null, processorTag, null, config);
-        try {
-            processor.execute(ingestDocument);
-            fail("remove field should have failed");
-        } catch (IllegalArgumentException e) {
-            assertThat(e.getMessage(), containsString("not present as part of path [" + fieldName + "]"));
-        }
+        assertThrows(
+            "field [" + fieldName + "] doesn't exist",
+            IllegalArgumentException.class,
+            () -> { processor.execute(ingestDocument); }
+        );
+
+        Map<String, Object> configWithEmptyField = new HashMap<>();
```

Comment:
Would be great to split this test case in two, testRemoveNonExistingField  / testRemoveEmptyField, but leaving it up to you